src/command/build.rs

```diff
@@ -71,21 +72,27 @@ impl Build {
     pub fn try_from_opts(build_opts: BuildOptions) -> Result<Self, Error> {
         let crate_path = set_crate_path(build_opts.path);
         let crate_name = manifest::get_crate_name(&crate_path)?;
+        let mode = match build_opts.mode.as_str() {
+            "no-install" => BuildMode::Noinstall,
+            "normal" => BuildMode::Normal,
```

Comment:
Is there a reason to have this if it always defaults to normal?

Reply:
Not really! I had included both cases for aesthetic reasons only, but I can clean that up.